versions-maven-plugin/src/main/java/org/codehaus/mojo/versions/DisplayParentUpdatesMojo.java

```diff
@@ -67,20 +153,13 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             logLine(false, "Parent project is part of the reactor.");
             return;
         }
-
-        String currentVersion = getProject().getParent().getVersion();
-        Artifact artifact = getHelper()
-                .createDependencyArtifact(DependencyBuilder.newBuilder()
-                        .withGroupId(getProject().getParent().getGroupId())
-                        .withArtifactId(getProject().getParent().getArtifactId())
-                        .withVersion(currentVersion)
-                        .withType("pom")
-                        .build());
-
+        String currentVersion = parentVersion == null ? getProject().getParent().getVersion() : parentVersion;
```

Comment:
or you could use Optional.ofNullable(parentVersion).orElse(getProject().getParent().getVersion())

but that's not really important ;)

Reply:
I would rather not do so, because `resolveTargetVersion` is expecting nullable String arguments anyway. But I renamed current to initialVersion, in order to be aligned with `UpdateParentMojo` .

Reply:
I meant, those two expressions are equivalent. Another idiom.